.github/workflows/native-binary-build-lib.profiler.yml

```diff
@@ -202,7 +206,7 @@ jobs:
          set OUTPUTDIR=../../release/lib/deployed/jdk16/windows
          del /q /s "%OUTPUTDIR%"
          rem   set up a Visual Studio command prompt for 32-bit development
-         call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars32.bat"
+         call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars32.bat" 10.0.20348.0
```

Comment:
"10.0.20348.0" this number looks very specific,  will this have to be updated often?

(thanks for fixing this btw!)

Reply:
If a SDK version is not specified the batch file will use the latest version. `10.0.20348.0` is the last Windows 10 SDK version before the Windows 11 SDK was released.

Name | Version Number | Build Number | Release Date
---|---|----|---
Windows Standalone SDK for Windows 10 | v10 | 10.0.10240.0 | 2015-07-29
Windows Standalone SDK for Windows 10, Version 1511 | v10 |  10.0.10586.212 | 2015-11-30
Windows Standalone SDK for Windows 10, Version 1607 | v10 | 10.0.14393.795 | 2016-08-02
Windows Standalone SDK for Windows 10 Creators Update, Version 1703 |v10 | 10.0.15063.0 | 2017-04-05
Windows 10 SDK for Fall Creators Update, version 1709 | v10 | 10.0.16299.15 | 2017-10-10
Windows 10 SDK for April 2018 Update, version 1803 | v10 | 10.0.17134.0 | 2018-05-08
Windows 10 SDK for October 2018 Update, version 1809 | v10 | 10.0.17763.0 | 2018-10-02
Windows 10 SDK for Windows 10, version 1903 | v10 | 10.0.18362.0 | 2019-05-21
Windows 10 SDK for Windows 10, version 2004 | v10 | 10.0.19041.0 | 2020-05-12
Windows 10 SDK for Windows 10, version 2004 | v10 | 10.0.19041.685 | 2020-12-16
Windows 10 SDK for Windows 10, version 21H1 | v10 | 10.0.20348.0 | 2021-05-25
Windows 10 SDK for Windows 11 | v10 |10.0.22000.194 | 2021-10-04

https://en.wikipedia.org/wiki/Microsoft_Windows_SDK

I don't think GitHub will remove the SDK from the image.
https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md
https://github.com/actions/virtual-environments/blob/main/images/win/Windows2022-Readme.md

Fixing the rc file to compile with the new SDK is a correct solution. I'm not sure what would need be changed to accomplish that. But in the meantime, this MR will allow the workflow to pass.